test/e2e/service_options_test.go

```diff
@@ -29,62 +28,55 @@ func TestServiceOptions(t *testing.T) {
-		test.serviceDescribeConcurrencyTarget(t, "hello", "300")
+	t.Run("update and validate service with concurrency limit", func(t *testing.T) {
+		test.serviceUpdate(t, "svc1", []string{"--concurrency-limit", "300"})
+		test.validateServiceConcurrencyLimit(t, "svc1", "300")
 	})
 
-	t.Run("update concurrency limit for hello service and returns no error", func(t *testing.T) {
-		test.serviceUpdate(t, "hello", []string{"--concurrency-limit", "300"})
+	t.Run("update concurrency options with invalid values for service", func(t *testing.T) {
+		test.serviceUpdateWithInvalidValue(t, "svc1", []string{"--concurrency-limit", "-1", "--concurrency-target", "0"})
 	})
 
-	t.Run("returns correct concurrency limit for hello service", func(t *testing.T) {
-		test.serviceDescribeConcurrencyLimit(t, "hello", "300")
+	t.Run("returns steady concurrency options for service", func(t *testing.T) {
+		test.validateServiceConcurrencyLimit(t, "svc1", "300")
+		test.validateServiceConcurrencyTarget(t, "svc1", "300")
 	})
 
-	t.Run("update concurrency options with invalid value for hello service and returns error", func(t *testing.T) {
-		test.serviceUpdateWithInvalidValue(t, "hello", []string{"--concurrency-limit", "-1", "--concurrency-target", "0"})
-	})
-
-	t.Run("returns steady concurrency options for hello service", func(t *testing.T) {
-		test.serviceDescribeConcurrencyLimit(t, "hello", "300")
-		test.serviceDescribeConcurrencyTarget(t, "hello", "300")
-	})
-
-	t.Run("delete hello service and returns no error", func(t *testing.T) {
-		test.serviceDelete(t, "hello")
+	t.Run("delete service", func(t *testing.T) {
+		test.serviceDelete(t, "svc1")
 	})
 }
 
-// Private
-
 func (test *e2eTest) serviceCreateWithOptions(t *testing.T, serviceName string, options []string) {
 	command := []string{"service", "create", serviceName, "--image", KnDefaultTestImage}
 	command = append(command, options...)
 	out, err := test.kn.RunWithOpts(command, runOpts{NoNamespace: false})
 	assert.NilError(t, err)
-
 	assert.Check(t, util.ContainsAll(out, "Service", serviceName, "successfully created in namespace", test.kn.namespace, "OK"))
 }
 
-func (test *e2eTest) serviceDescribeConcurrencyLimit(t *testing.T, serviceName, concurrencyLimit string) {
-	out, err := test.kn.RunWithOpts([]string{"service", "describe", serviceName}, runOpts{NoNamespace: false})
+func (test *e2eTest) validateServiceConcurrencyLimit(t *testing.T, serviceName, concurrencyLimit string) {
+	//jsonpath := "jsonpath={.items[0].spec.template.spec.containerConcurrency}"
+	//debug
+	out, err := test.kn.RunWithOpts([]string{"service", "describe", serviceName}, runOpts{})
+	//debug
+	assert.Equal(t, "foo", out)
 	assert.NilError(t, err)
-
-	expectedOutput := fmt.Sprintf("containerConcurrency: %s", concurrencyLimit)
-	assert.Check(t, util.ContainsAll(out, expectedOutput))
+	//out, err := test.kn.RunWithOpts([]string{"service", "list", serviceName, "-o", jsonpath}, runOpts{})
```

Comment:
Can we remove commented code?

Reply:
Sure, this was added as debug commit to check the reason for continuous failure of e2e job `pull-knative-client-integration-tests-latest-release` we've, please check comment https://github.com/knative/client/pull/302#issuecomment-515297563 ,  I'll replace the debug commit with a fix. Thanks!